lib/std/os/windows/user32.zig

```diff
@@ -1,104 +1,100 @@
 usingnamespace @import("bits.zig");
 
-// PM
+pub const WM = enum(u16) {
```

Comment:
Looks like this should be `UINT` rather than `u16`. Also the enumeration namespace is shared with a whole host of others: https://docs.microsoft.com/en-au/windows/win32/winmsg/about-messages-and-message-queues

So I think this should be 
```
pub const MessageTypes = enum(UINT) {
    WM_NULL = 0x0000,
    WM_CREATE = 0x0001,
    WM_DESTROY = 0x0002,
    WM_MOVE = 0x0003,
    WM_SIZE = 0x0005,

     ..... etc

    _,
}
```

You could even update `WindowProc` to change the `uMsg` argument to take that `MessageTypes`.

Reply:
Should the other enum's also be UINT?

Reply:
For `WNDPROC` if I changed the `uMsg` type to `MessageTypes` would it pose any issue? Is the conversion implicit from the caller passing a `UINT` to it becoming the enum?

Reply:
Since the conversion between enums and integers is purely to appease the Zig type system, and the underlying bytes and size are the same, the right bits will be in the right places as long as the calling convention treats the values the same as integers.  The enum will have to be marked as `extern enum(UINT)` to guarantee that it is compatible with the C ABI, in which enums and integers obey the same rules.